http/src/client/mod.rs

```diff
@@ -838,6 +838,21 @@ impl Client {
         GetMember::new(self, guild_id, user_id)
     }
 
+    /// Add a user to the guild.
+    ///
+    /// You must have an [access token] for this user with the `guilds.join`
+    /// scope.
+    ///
+    /// [access token]: https://discord.com/developers/docs/topics/oauth2
```

Comment:
The docs here should match the module-level docs, I'll leave it to you to decide which to use.